Kajabi/sage-lib
# Button: React documentation updates

## Description

Update the React documentation to more clearly outline the options available to `<Button>`

## Screenshots from button
|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2023-08-23 at 2 56 31 PM](https://github.com/Kajabi/sage-lib/assets/24237393/594a2ed1-ed2a-4fd2-ad2f-aaee5d19d5b6) |![Screen Shot 2023-08-23 at 2 57 21 PM](https://github.com/Kajabi/sage-lib/assets/24237393/a0f58b72-4235-4df8-9c5e-3a148fdbfb5a)|
| ![Screen Shot 2023-08-23 at 2 56 45 PM](https://github.com/Kajabi/sage-lib/assets/24237393/d67f2061-47ce-4bf6-be18-f48fd5c52393) | ![Screen Shot 2023-08-23 at 2 57 28 PM](https://github.com/Kajabi/sage-lib/assets/24237393/e6d260be-4f35-4274-997b-25892607657c) |


## Screenshots from toolbar editor
|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2023-08-23 at 2 57 14 PM](https://github.com/Kajabi/sage-lib/assets/24237393/7639289d-b161-4e82-9222-933173d2a279) | ![Screen Shot 2023-08-23 at 3 13 24 PM](https://github.com/Kajabi/sage-lib/assets/24237393/b5782093-e936-42a9-98dd-5a7da69b12de) |


## Testing in `sage-lib`
1. Navigate to [button](http://localhost:4100/?path=/docs/sage-button--primary) 
2. Verify the Button documentation is fully representative of the button component
3. Navigate to [toolbar editor](http://localhost:4100/?path=/docs/sage-toolbareditor--default)
4. Verify the toolbar editor documentation for the buttons is accurate

## Testing in `kajabi-products`
1. (**LOW**) No props were altered, documentation changes only
